components/claim/claimable-rewards.tsx

```diff
@@ -184,6 +184,12 @@ const ClaimableRewards: React.FC<ClaimableRewardsProps> = ({ isAdmin }) => {
     return null;
   }
 
+  const meeeee = mappedData.map((item) => {
```

Comment:
what is this variable name exactly

Reply:
testing stuff :( I removed it now, sorry!